### PR TITLE
ZenML Hub Example

### PR DESCRIPTION
**Pull request by fa9r**

### CODE REVIEW

File: hub-example/README.md
Comment (strickvl) [inline hub-example/README.md:1]: This readme needs a link to the launch blog post, and also link to the docs (in a 'getting started with the hub' section, maybe, towards the bottom?)

**Review by strickvl — CHANGES_REQUESTED** [review]
Also a typo to fix in the notebook...

**Comment by fa9r** [comment]
> Also a typo to fix in the notebook...

where? 😄

File: hub-example/README.md
Comment (fa9r) [reply hub-example/README.md:1]: I had all this in the notebook, just copied it over now to a "Next Steps" section at the end, what do you think?

**Review by strickvl — APPROVED** [review]

**Review by strickvl — APPROVED** [review]

**Pull request merged**